lib/logstash/outputs/elasticsearch/template_manager.rb

```diff
@@ -30,7 +30,8 @@ def self.install(client, template_name, template, template_overwrite)
     end
 
     def self.default_template_path(es_major_version)
-      default_template_name = "elasticsearch-template-es#{es_major_version}x.json"
+      es_major_version == "1" ? template_version = "2" : template_version = es_major_version
```

Comment:
I think you meant this

```ruby
template_version = es_major_version == "1" ?  "2" :  es_major_version
```

I was a bit confused with the assignment in the ternary ops.